### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Footer_ColorContrast: The color contrast of the link "134,941" in the footer section when focus is on it is 3.818:1 which is less than 4.5:1

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/vscode-docs#5529]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win10;#FTP;#DesktopWeb;#A11ySev2;#WCAG1.4.3;#ColorContrast;#ChromiumEdge;

## Environment Details:
Application Name: Visual Studio Code Services 
URL: https://code.visualstudio.com
Windows Version: Win10
ChromiumEdge Version: Version 104.0.1293.47(Official build)(64-bit)

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com
3. Navigate to footer section.
4. Tab till link "134,941" and check color contrast for it.
5. Observe that color contrast for link "1,34,941" is 3.818:1 which is less than 4.5:1

## Actual:
The color contrast of the link "134,941" in the footer section when focus is on it is 3.818:1 which is less than 4.5:1

## Expected:
The color contrast of the link "134,941" in the footer section when focus is on it should be equal to or greater than 4.5:1.

## User Impact:
Low Vision users will face difficulty in viewing the control when focus is on the link.

## Attachment
![5529_A11y_Visual Studio Code Services](https://user-images.githubusercontent.com/93374043/183882621-4a0fe1b8-03d2-44df-8b1a-c7fbdbfb0ef3.png)

**Comment by gregvanl** [comment]
External as we don't control the GitHub stargazer UI, we just place the control in the footer
Closing as no "Resolved" state for GitHub issues

**Comment by Yashi1201** [comment]
@gregvanl Could you please provide the vso and correct area path so that we can log an external bug to them.

**Comment by gregvanl** [comment]
Looks like we took a snapshot of that control source so we can tune the contrast.

**Comment by gregvanl** [comment]
Fixed and closing since no "Resolved" state for GitHub issues.

**Comment by Yashi1201** [comment]
Closing the bug as above issue is not repro.
![5529_closing](https://user-images.githubusercontent.com/93374043/192705845-ae2358c5-0299-4521-9b3b-0937b1b8d052.png)
